### PR TITLE
Allow triagers to close PRs if the PR is of very low quality

### PR DESCRIPTION
**Pull request by AlexWaygood**
As discussed on the CPython core-dev discord channel, there are several issues with the current policy that states that triagers should never close PRs:
- Github's setup allows members of the triage team to close PRs, regardless of the policy stated in the devguide.
- This policy isn't consistently followed by all members of the triage team. Some triagers occasionally close PRs (I have closed one or two myself in situations where it seemed abundantly clear that there was no prospect of the PR being merged).
- This policy isn't really enforced in any way if triagers do close PRs. This lack of enforcement seems to be an implicit endorsement of the idea that it is okay for triagers to close PRs in some circumstances.
- It feels silly to have to bother a core dev with a request for a PR to be closed if it's self-evident that it should be closed (e.g. if the PR makes changes to a deprecated module, makes solely cosmetic changes to a module, or simply has a 0% chance of ever being merged)
- Changing this policy will help towards the goal of reducing the CPython PR backlog

### CODE REVIEW

File: triaging.rst
Comment (iritkatriel) [inline triaging.rst:46]: also:  PRs which are no longer necessary because the issue had been fixed or made irrelevant by other PRs merged since this one was created.

File: triaging.rst
Comment (iritkatriel) [inline triaging.rst:48]: I wonder if we can reword this so that it's clearer what is meant. Often we get a PR with no unit test, and that makes it far below the threshold for making the change, but we don't close it, we ask for a unit test.

Comment (AlexWaygood) [reply triaging.rst:48]: I was thinking of PRs like, for example, the recently closed 32408 (deliberately not linking to it directly). That PR isn't a spammy PR — the author clearly has the best of intentions, has signed the CLA, and has even opened a BPO issue. But, with the best will in the world, that PR had a 0% chance of ever being merged.

How do we describe a PR like that?

Comment (iritkatriel) [reply triaging.rst:48]: I think the issue here is not so much the quality of the PR as much as the quality of the whole idea (the quality of the bpo). We have a lot of open issues with bad enhancement suggestion. How to weed them out is one of the questions I hope we can discuss at the language summit.

Comment (AlexWaygood) [reply triaging.rst:48]: Yes. It's easy to say "this is not a bug". It's much more difficult to say "I'm sorry, you may think that this is a good idea, but it just isn't, and we're never realistically going to implement it".

Comment (iritkatriel) [reply triaging.rst:48]: Agreed. Maybe we leave such things out of the dev guide and just focus on the really obvious ones.

Comment (AlexWaygood) [reply triaging.rst:48]: Sure. I'll make the change.

File: triaging.rst
Comment (terryjreedy) [reply triaging.rst:46]: Non-coredev triagers can close issues in certain circumstances (which is a separate discussion) and when then do, they should close any associated PR.  (Same generally true for coredevs.)  You should say at least that much and remove any existing blanket permission denial.  Closing PRs for legitimate open issues is trickier.  I agree with sticking with the most obvious cases and not trying to be complete.  If in doubt, asking for a second opinion is not a bad thing.

File: triaging.rst
Comment (JelleZijlstra) [reply triaging.rst]: Would it be worth adding something like "if in doubt, leave it to a core dev"?

Comment (AlexWaygood) [reply triaging.rst]: How's it look after f2166e7?

File: triaging.rst
Comment (hugovk) [inline triaging.rst:46]: A bit more concise:
```suggestion
revisions were made to the PR. This includes (but is not limited to) the
following:
```

**Review by JelleZijlstra — APPROVED** [review]

**Comment by AlexWaygood** [comment]
Thanks all!

**Pull request merged**
